diff --git a/node_modules/@noble/hashes/_sha2.js b/node_modules/@noble/hashes/_sha2.js
index 645e58a..479287b 100644
--- a/node_modules/@noble/hashes/_sha2.js
+++ b/node_modules/@noble/hashes/_sha2.js
@@ -9,7 +9,7 @@ function setBigUint64(view, byteOffset, value, isLE) {
         return view.setBigUint64(byteOffset, value, isLE);
     const _32n = BigInt(32);
     const _u32_max = BigInt(0xffffffff);
-    const wh = Number((value >> _32n) & _u32_max);
+    const wh = Number((value.shiftRight(_32n)) & _u32_max);
     const wl = Number(value & _u32_max);
     const h = isLE ? 4 : 0;
     const l = isLE ? 0 : 4;
diff --git a/node_modules/@noble/hashes/esm/_sha2.js b/node_modules/@noble/hashes/esm/_sha2.js
index 026dcf1..85cfa1e 100644
--- a/node_modules/@noble/hashes/esm/_sha2.js
+++ b/node_modules/@noble/hashes/esm/_sha2.js
@@ -6,7 +6,7 @@ function setBigUint64(view, byteOffset, value, isLE) {
         return view.setBigUint64(byteOffset, value, isLE);
     const _32n = BigInt(32);
     const _u32_max = BigInt(0xffffffff);
-    const wh = Number((value >> _32n) & _u32_max);
+    const wh = Number((value.shiftRight(_32n)) & _u32_max);
     const wl = Number(value & _u32_max);
     const h = isLE ? 4 : 0;
     const l = isLE ? 0 : 4;
