diff --git a/node_modules/@testing-library/react-native/build/render.d.ts b/node_modules/@testing-library/react-native/build/render.d.ts
index b7263db..40be56b 100644
--- a/node_modules/@testing-library/react-native/build/render.d.ts
+++ b/node_modules/@testing-library/react-native/build/render.d.ts
@@ -1,6 +1,6 @@
 import TestRenderer from 'react-test-renderer';
 import * as React from 'react';
-declare type Options = {
+export declare type Options = {
     wrapper?: React.ComponentType<any>;
     createNodeMock?: (element: React.ReactElement) => any;
 };
