diff --git a/node_modules/react-native-circular-progress/index.d.ts b/node_modules/react-native-circular-progress/index.d.ts
index 4037714..f73962b 100644
--- a/node_modules/react-native-circular-progress/index.d.ts
+++ b/node_modules/react-native-circular-progress/index.d.ts
@@ -1,11 +1,11 @@
 declare module 'react-native-circular-progress' {
   import * as React from 'react';
   import {
-    Animated,
-    EasingFunction,
-    StyleProp,
-    ViewStyle
-  } from 'react-native';
+  Animated, ColorValue,
+  EasingFunction,
+  StyleProp,
+  ViewStyle,
+} from "react-native";
 
   export interface AnimatedCircularProgressProps {
     /**
@@ -43,7 +43,7 @@ declare module 'react-native-circular-progress' {
      * @type {string}
      * @default 'black'
      */
-    tintColor?: string;
+    tintColor?: ColorValue | string;
 
     /**
      * Change the fill color from tintColor to tintColorSecondary as animation progresses.
@@ -51,7 +51,7 @@ declare module 'react-native-circular-progress' {
      * @type {string}
      * @default 'undefined'
      */
-    tintColorSecondary?: string;
+    tintColorSecondary?: ColorValue | string;
 
     /**
      * Current progress / tint transparency
