diff --git a/node_modules/react-native-image-crop-picker/index.d.ts b/node_modules/react-native-image-crop-picker/index.d.ts
index d3e1fd4..7794f19 100644
--- a/node_modules/react-native-image-crop-picker/index.d.ts
+++ b/node_modules/react-native-image-crop-picker/index.d.ts
@@ -340,6 +340,14 @@ declare module "react-native-image-crop-picker" {
          * @default Android: 1, iOS: 0.8
          */
         compressImageQuality?: number;
+      
+        /**
+         * When cropping image, uses the dimensions of the cropRect for the resulting image.
+         *
+         * @platform iOS only
+         * @default false
+         */
+        useCroppedDimensions?: boolean;
     }
 
     type CropperOptions = ImageOptions & {
diff --git a/node_modules/react-native-image-crop-picker/ios/src/ImageCropPicker.m b/node_modules/react-native-image-crop-picker/ios/src/ImageCropPicker.m
index 9f20973..6f3279f 100644
--- a/node_modules/react-native-image-crop-picker/ios/src/ImageCropPicker.m
+++ b/node_modules/react-native-image-crop-picker/ios/src/ImageCropPicker.m
@@ -72,7 +72,8 @@ - (instancetype)init
             @"sortOrder": @"none",
             @"cropperCancelText": @"Cancel",
             @"cropperChooseText": @"Choose",
-            @"cropperRotateButtonsHidden": @NO
+            @"cropperRotateButtonsHidden": @NO,
+            @"useCroppedDimensions": @NO
         };
         self.compression = [[Compression alloc] init];
     }
@@ -799,11 +800,16 @@ - (void)dismissCropper:(UIViewController *)controller selectionDone:(BOOL)select
 - (void)imageCropViewController:(UIViewController *)controller
                    didCropImage:(UIImage *)croppedImage
                   usingCropRect:(CGRect)cropRect {
+    CGSize desiredImageSize;
     
-    // we have correct rect, but not correct dimensions
-    // so resize image
-    CGSize desiredImageSize = CGSizeMake([[[self options] objectForKey:@"width"] intValue],
-                                         [[[self options] objectForKey:@"height"] intValue]);
+    if ([[[self options] objectForKey:@"useCroppedDimensions"] boolValue]) {
+        desiredImageSize = cropRect.size;
+    } else {
+        // we have correct rect, but not correct dimensions
+        // so resize image
+        desiredImageSize = CGSizeMake([[[self options] objectForKey:@"width"] intValue],
+                                             [[[self options] objectForKey:@"height"] intValue]);
+    }
     
     UIImage *resizedImage = [croppedImage resizedImageToFitInSize:desiredImageSize scaleIfSmaller:YES];
     ImageResult *imageResult = [self.compression compressImage:resizedImage withOptions:self.options];
