diff --git a/node_modules/react-native-toast-notifications/src/hook/provider.tsx b/node_modules/react-native-toast-notifications/src/hook/provider.tsx
index 10d238e..4980492 100644
--- a/node_modules/react-native-toast-notifications/src/hook/provider.tsx
+++ b/node_modules/react-native-toast-notifications/src/hook/provider.tsx
@@ -11,8 +11,9 @@ const ToastProvider: FC<PropsWithChildren> = ({ children, ...props }) => {
   const [refState, setRefState] = useState({});
 
   useEffect(() => {
-    setRefState(toastRef.current as any);
-  }, []);
+    if(toastRef.current)
+      setRefState(toastRef.current as any);
+  }, [toastRef.current]);
 
   return (
     <ToastContext.Provider value={refState as any}>
diff --git a/node_modules/react-native-toast-notifications/src/toast.tsx b/node_modules/react-native-toast-notifications/src/toast.tsx
index 7b07fc3..a7c32ef 100644
--- a/node_modules/react-native-toast-notifications/src/toast.tsx
+++ b/node_modules/react-native-toast-notifications/src/toast.tsx
@@ -101,6 +101,11 @@ export interface ToastOptions {
    */
   onPress?(id: string): void;
 
+  /**
+   * Callback for when the retry button of the toast is pressed.
+   */
+  onPressRetry?(id: string): void;
+
   /**
    * Execute event after toast is closed
    */
@@ -146,7 +151,8 @@ const Toast: FC<ToastProps> = (props) => {
     placement,
     swipeEnabled,
     onPress,
-  } = props;
+    onPressRetry,
+} = props;
 
   const containerRef = useRef<View>(null);
   const [animation] = useState(new Animated.Value(0));
